compiler/one-cmds/onelib/constant.py

```diff
@@ -48,6 +48,7 @@ class CONSTANT:
         'fuse_instnorm',
         'fuse_prelu',
         'fuse_gelu',
+        'fuse_rmsnorm',
```

Comment:
Could you exclude this option in O1? Our backend compiler is not ready for this change.

Reply:
Oh, I couldn't think of that. 
I'll update it :)